.github/workflows/validation.yaml

```
@@ -43,3 +43,6 @@ jobs:
         run: |
           go test ./... -coverprofile coverage.out -covermode count -tags=unit
           go tool cover -func coverage.out
+      - name: Build Integration Tests
+        run: |
+          go test --tags=integration ./integration -c -o integration/sb-test
```

align with https://github.com/eclipse-kanto/suite-connector/blob/main/.github/workflows/validation.yaml
```suggestion
go test --tags=integration ./integration -c -o integration/bin/sb-test
```